lib/Mojolicious/Plugin/DefaultHelpers.pm

```diff
@@ -41,6 +42,9 @@ sub register {
   $app->helper(dumper  => sub { shift; dumper @_ });
   $app->helper(include => sub { shift->render_to_string(@_) });
 
+  $app->helper('proxy.get_p'   => sub { _proxy_method_p('GET', @_) });
```

Comment:
Why not just do this?

```
$app->helper('proxy.get_p' => sub { _proxy_start_p($_[0], shift->ua->build_tx(get => @_)) };
```

Reply:
Because i might want to add `proxy->post_p` before the PR is merged.